### ISSUE TITLE
[ENHANCE] Sonic Redstone Function

### ISSUE THREAD

**Issue by AlienXtream** [DarkholmeTenk/TardisModIssues#375]
@DarkholmeTenk Ok, just hear me out here. you said that Redstone would be a pain in the neck BUT what about a sonic "lever" and "button" putting a lever or button in a temporal lab will result in a Sonic Toggle and  sonic Pulsar respectively. the idea of them would be if the sonic is activated in range (16 blocks?) they will activate a redstone signal, sorta like short range wireless redstone. 
new sonic mode "Redstone" (color red with redstone particles?) would activate on right clicking.
reconfigure mode can set pulse length and new permissions to allow other sonics to activate your redstone devices

**Comment by FoxPotato** [comment]
but wouldn't just being able to activate one block be easier then?

**Comment by AlienXtream** [comment]
touche

**Comment by grodax** [comment]
There plugin allows you to do it

**Comment by AlienXtream** [comment]
There ale also item teleportation mods(like ender chests) fluid storage and wireless power transfer just ti name a few. If we all did totalt original thingsbwebwould quickly run out of things to make. The beuaty of a concept is there can be many ways to execute it.

**Comment by FoxPotato** [comment]
I agree, but also if we all copied each other. it would become quite boring ;p

**Comment by AlienXtream** [comment]
True. As with anything in our universe balance is crucial. You need the right level of inspiration and uniqueness. For instance, darks no gui philosophy for the mod makes for an interesting challenge both for users AND for dark (regurding coding the features)
